images/component-server-image/pom.xml

```diff
@@ -163,7 +161,7 @@
             </goals>
             <phase>prepare-package</phase>
             <configuration>
-              <outputDirectory>${additionalDockerFiles}/opt/talend/component-kit/conf</outputDirectory>
+              <outputDirectory>${docker.image.extra.files}/opt/talend/component-kit/conf</outputDirectory>
```

Comment:
Don't see where you get this propertly. Where is it declared?

Reply:
Here: https://github.com/Talend/component-runtime/blob/undx/TCOMP-2367_jib/images/pom.xml#L60